### PR TITLE
Update setting custom target name doc

### PR DESCRIPTION
**Pull request by runleonarun**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ where created_at > date_trunc('month', current_date)
 
 To set a custom target name for a job in dbt Cloud, configure the "Target Name" field for your job in the Job Settings page.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md]:
```suggestion
To set a custom target name for a job in dbt Cloud, configure the **Target Name** field for your job in the Job Settings page.
```

Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md]: sugg: bold UI text field names

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ where created_at > date_trunc('month', current_date)
...
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/60f3fa2-Screen_Shot_2019-02-08_at_10.33.20_PM.png" title="Overriding the target name to 'prod'"/>
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/jobs-settings-target-name.png" title="Overriding the target name to 'prod'"/>
 
 ## dbt Cloud IDE
-When developing in dbt Cloud, you can set a custom target name in your development credentials. Head to your account (via your profile image in the top right hand corner), select the project under "Credentials" and update the target name.
+When developing in dbt Cloud, you can set a custom target name in your development credentials. Head to your account (via the gear in the top right hand corner), select the project under "Credentials" and update the target name.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md]:
```suggestion
When developing in dbt Cloud, you can set a custom target name in your development credentials. Go to your account (from the gear menu in the top right hand corner), select the project under **Credentials**, and update the target name.
```

Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-setting-a-custom-target-name.md]: suggestion: minor nits for more clarity like removing potential USAn colloquialism (head), avoid Latin abbreviation, and used an Oxford comma in the series

**Review by nghi-ly — CHANGES_REQUESTED** [review]
@runleonarun : looks great! left minor nits for more clarity to the reader

**Comment by runleonarun** [comment]
Thanks @nghi-ly! Committed those suggested changes.

**Review by nghi-ly — APPROVED** [review]
great, tysm for processing the suggs! will approve and merge this!

**Pull request merged**